### ISSUE TITLE
FR: Allow all Command Line Options to be utilised in the Command File

### ISSUE THREAD

**Issue by smithjw** [bartreardon/swiftDialog#124]
Unsure of the effort required, but it would be great if any functionality that can be triggered via Command Line options when Dialog is launched, could also be triggered in a currently running Dialog window via the Command File

**Comment by dan-snelson** [comment]
For example, `--overlayicon` doesn't appear to be listed in [Updating Dialog with new content (v1.9.0)](https://github.com/bartreardon/swiftDialog/wiki/Updating-Dialog-with-new-content--(v1.9.0)).

**Comment by scriptingosx** [comment]
I don't know if this is possible or if it should be a new issue, but I would like to replace the entire info area using the command file, i.e, I have a list of checkboxes, then I replace that with a list of text fields for user entry, then I show an list of items

**Comment by bartreardon** [comment]
@scriptingosx  yeah that's more than doable now. I'll need to add the code to action the update commands but that shouldn't be a problem to do. The only thing I'll need to consider is what to do with the details entered. At the moment everything would be stored into a dictionary and then when the dialog exits you'd get a dump of everything. i.e. no in-line processing of user input (display a checkbox and use the result to influence the next thing you show...that type of thing)

**Comment by macvfx** [comment]
Adding votes for —button1shellaction to work in the command file along with the other good stuff. Would love the button run a script. Or my case open a file. Maybe a separate issue is needed.

**Comment by bartreardon** [comment]
closing this one as it's old and the number of arguments is at the point where this specific type of implementation would be done differently anyway
